app/build.gradle.kts

```diff
@@ -8,6 +8,9 @@ plugins {
 
     id("com.google.gms.google-services")
     id("com.google.firebase.crashlytics")
+
+    id("app.cash.paparazzi")
```

Comment:
Remove this one - you don't need it? If you need it you must define Paparazzi not as plugin but how Room is defined:
1. As a library in the version catalog
2. implementation in buildSrc
3. Only after the above you can use it like that

@Vivekban this is the problem